frontend/src/renderer/components/ImageImporter.svelte

```diff
@@ -0,0 +1,148 @@
+<script lang="ts">
+    import { FilePlusIcon } from "svelte-feather-icons";
+    import { processState, sendMessage, messageStore } from "@util/stores";
+    import { forEach, find } from "lodash";
+    import ImageBubble from "@components/Process/ImageBubble.svelte";
+    import Dropzone from "svelte-file-dropzone";
+    export let label = "Select Files";
+    export let icon = FilePlusIcon;
+    export let largeText = false;
+    let filePaths = [];
+    let files = {
+        accepted: [],
+        rejected: []
+    };
+    let MAX_FILES = 8
+
+    function getThumbnails() {
+        console.log("Getting Thumbnails");
+        $processState.thumbnailID = Math.floor(Math.random() * 999999999);
+        let msg = {
+            RequestID: $processState.thumbnailID,
+            RequestType: "Thumbnails",
+            RequestData: {
+                names: filePaths,
+            },
+        };
+        console.log(msg);
+        sendMessage(JSON.stringify(msg));
+        filePaths = [];
+    }
+
+    $: if (filePaths?.length > 0) {
+        console.log("Fetching Thumbnails");
+        getThumbnails();
+    }
+
+    $: if (
+        $processState.imageFilePaths.length >= 6 &&
+        !$processState.completedTabs[0]
+    ) {
+        $processState.completedTabs[0] = true;
+    }
+
+    function handleFilesSelect(e) {
+        console.log("files")
+        if (files.accepted.length < MAX_FILES){
+            files.accepted = [];
+            console.log("accepted")
+            const {acceptedFiles, fileRejections} = e.detail;
+            files.accepted = [...files.accepted, ...acceptedFiles];
+            files.rejected = [...files.rejected, ...fileRejections];
+
+            forEach(files.accepted, (f) => {
+                if (!find(filePaths, {id: f.path, name: f.name})) {
+                    filePaths.push({
+                        id: f.path,
+                        name: f.path
+                    });
+                }
+            });
+            $processState.imageFilePaths = [...filePaths];
+        }
+    }
+</script>
+
+<main>
+    <br>
+    <Dropzone
+            on:drop={handleFilesSelect}
+
+            containerStyles="flex: 1;
+                            display: flex;
+                            flex-direction: column;
+                            align-items: center;
+                            padding: 20px;
+                            border-width: 2px;
+                            border-radius: 2px;
+                            border-color: #eeeeee;
+                            border-style: dashed;
+                            background-color: #1D1C1E;
+                            color: #bdbdbd;
+                            outline: none;
+                            transition: border 0.24s ease-in-out;
+                            background-color: #1D1C1E;
+                            border-radius: 10px;
+                            font-size: 30px;
+                            text-align: center;"
```

Comment:
I like all the css, but please move it from inline to somewhere else. If it can't be moved into the dedicated styles block because of the weird dropzone library, could you try assigning all this to a string and passing that in?

Reply:
So, I do not believe I can have it in a style block.  I have tried putting it in a string, but the button is no longer centered-aligned.